### PR TITLE
Bt mbta commands

### PR DESCRIPTION
**Pull request by brimatteng**
Created a Kevinbot commmand to return the next time of trains departing from Kendall Sq stop
- Still needs logic refactored and abstracted 
- returns both a north and southbound train
- Requires an API key to access the mbta "realtime" api

### CODE REVIEW

File: commands/nextredline.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    if next_north_train_from_braintree < next_north_train_from_ashmont
+      next_north_train_from_braintree
+    else
+      next_north_train_from_ashmont
+    end
+  end
+
+  ## This is only the alewife -> ashmont line
+  def next_south_train_to_ashmont
+    Time.at(response_body["mode"][0]["route"][0]["direction"][0]["trip"][0]["pre_dt"].to_i)
```
Comment (edwardloveall) [inline commands/nextredline.rb]: I like that you made the `response_body` and `response` methods. I'd make another for the route. Something like `ashmont_south`, `braintree_north`

Comment (edwardloveall) [inline commands/nextredline.rb]: It could help get rid of the comments

**Comment by edwardloveall** [comment]
Please rewrite the overall comment for posterity

**Comment by Benuuu** [comment]
@brimatteng @edwardloveall what's the state of this PR?  Looks like there's at least one commit that shouldn't be in here e27e32d

**Comment by edwardloveall** [comment]
Probably need to be tested. If it works, you can merge or rebase if you want. ¯\_(ツ)_/¯

**Comment by brimatteng** [comment]
I left this in a working state. As long as the realtime api for MBTA hasn't changed, it should work. 
It takes a long time to get a token for it if you haven't already. It took me a week

**Comment by brimatteng** [comment]
I'm gonna close this because I'm probably never going to finish this

**Pull request closed without merging**